tests/httpwrappers/tests.py

```diff
@@ -597,6 +598,12 @@ def test_json_response_passing_arguments_to_json_dumps(self):
         response = JsonResponse({'foo': 'bar'}, json_dumps_params={'indent': 2})
         self.assertEqual(response.content.decode(), '{\n  "foo": "bar"\n}')
 
+    def test_json_response_callable_bool(self):
+        response = JsonResponse(CallableTrue, safe=False)
+        self.assertEqual(json.loads(response.content.decode()), bool(CallableTrue))
```

Comment:
The test would be better located in `tests/serializers/test_json.py`, I think. For the assertion, `self.assertIs(... True)` seems more straightforward than `bool(CallableTrue)`.